python_modules/libraries/dagster-pagerduty/dagster_pagerduty/resources.py

```diff
@@ -16,8 +20,12 @@ class PagerDutyService:
     for documentation and more information.
     """
 
-    def __init__(self, routing_key: str):
-        self.routing_key = routing_key
+    routing_key: str = PyField(
+        ...,
+        description="""The routing key provisions access to your PagerDuty service. You
+                    will need to include the integration key for your new integration, as a
+                    routing_key in the event payload.""",
+    )
```

Comment:
should this be something like this:
```suggestion
        description=(
            "The routing key provisions access to your PagerDuty service. You",
            "will need to include the integration key for your new integration, as a"
            "routing_key in the event payload."
        ),
    )
```
i haven't checked out the code in my local but i wonder if the triple quotes with indentation would result in a lot whitespaces showing up in the UI?